src/metldata/builtin_transformations/common/instruction.py

```diff
@@ -17,7 +17,8 @@
 
 from typing import Protocol, TypeVar
 
-from metldata.builtin_transformations.common import NewContentSchemaPath, SourcePath
+from metldata.builtin_transformations.common import SourcePath
+from metldata.builtin_transformations.common.contentschema import NewContentSchemaPath
```

Comment:
Both of them can be inherited directly from `common`, can't they? 
```suggestion
from metldata.builtin_transformations.common import NewContentSchemaPath, SourcePath
```